### ISSUE TITLE
[Feature Request] Add an about page

### ISSUE THREAD

**Issue by toby63** [lxc/linuxcontainers.org#438]
On this page you could describe the usual things:
-who you are
-what you do
etc.

**Comment by stgraber** [comment]
Yeah, I guess we can link it under the `Home` menu header and probably start with what we have at the top of the frontpage as content and expand from there.

**Comment by toby63** [comment]
Could we add photos of you from your GitHub profiles?
(I guess each team member should be asked)

**Comment by stgraber** [comment]
Yeah, we could probably at least add something about the maintainers and link to https://github.com/orgs/lxc/people for the rest or something like that.

**Comment by stgraber** [comment]
Would be neat if we could automate something that scrapes the Github info for those accounts to generate that page, it'd save us from having to update or from forgetting someone :)

**Comment by toby63** [comment]
> Would be neat if we could automate something that scrapes the Github info for those accounts to generate that page, it'd save us from having to update or from forgetting someone :)

:thinking: So I guess you want to use something like the GraphQL API?
It seems that many information pieces can be extracted, like Name, Bio, Avatar picture etc.: https://docs.github.com/en/graphql/reference/objects#user

**Comment by stgraber** [comment]
Their graphql API is way overkill for this.

In our case, we can probably just hit https://api.github.com/orgs/lxc/public_members and then fetch the profile photo and bio from https://api.github.com/users/stgraber. It's not really telling us what specific project they're associated with though so we'd probably just need to keep a map on the site for that purpose, still far less work than having to duplicate all the bio data.

Actually I suspect that it will be easier to just maintain a list of members and a one line description of their role in the project and then fetch the bio and photo from Github.

**Comment by toby63** [comment]
Interesting :thinking:, that is indeed an easier method.

> Actually I suspect that it will be easier to just maintain a list of members and a one line description of their role in the project and then fetch the bio and photo from Github.

That is probably true.
Nonetheless some changes might be necessary, e.g.:
- some members seem to use non-face profile pictures (also the (probably alcoholic) bottle on @hallyn's page might be something you don't want, I suspect)
- many members don't really have a good bio or no bio at all

So either they would change their Profiles or some changes have to be added manually here.

**Comment by stgraber** [comment]
Hmm, yeah, most people not having anything in their bio isn't going to help much ;)
